podvm/Dockerfile.podvm.centos

```diff
@@ -11,10 +11,12 @@ FROM ${BUILDER_IMG} AS podvm_builder
 ARG CLOUD_PROVIDER=libvirt
 ARG PODVM_DISTRO=centos
 ARG AA_KBC=offline_fs_kbc
+ARG QEMU_BINARY="/usr/libexec/qemu-kvm"
```

Comment:
Without having to add an extra variable to pass, you could probably just include the logic in the Makefile to first search /usr/libexec/qemu-kvm followed by /usr/bin/qemu-system-x86_64 for the the qemu binary.

Reply:
right, thing is we are now supports different architectures also, I'll need to think how to handle this